### ISSUE TITLE
azurerm_app_service not able to deploy php on windows

### ISSUE THREAD

**Issue by mleziva** [hashicorp/terraform-provider-azurerm#8821]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Affected Resource(s)

* `azurerm_app_service`

### Terraform Configuration Files

```
provider "azurerm" {
    version         = "~> 2.27.0"
    features {}
}

resource "azurerm_resource_group" "example" {
  name     = "ausmlepoc"
  location = "eastus2"
}

resource "azurerm_app_service_plan" "appserviceplan" {
  name                = "appserviceplan01"
  location            = "eastus2"
  resource_group_name = azurerm_resource_group.example.name
  kind = "Windows"
  reserved = false
  is_xenon = false
  sku {
    tier = "PremiumV2"
    size = "P1v2"
  }
}


resource "azurerm_app_service" "example" {
  name                = "nbac-ausmle-poc"
  location            = azurerm_resource_group.example.location
  resource_group_name = azurerm_resource_group.example.name
  app_service_plan_id = azurerm_app_service_plan.appserviceplan.id

  site_config {
    windows_fx_version               = "PHP|7.3"
    always_on                       = true
    php_version                     = "7.3"
  }
}
```

### Expected Behavior

App service should be deployed as PHP application using version 7.3 on windows

### Actual Behavior

App service is deployed as .NET application on windows

### Steps to Reproduce

1. `terraform apply`

**Comment by njuCZ** [comment]
@mleziva thanks for pointing out this issue. I have fired an upstream issue https://github.com/Azure/azure-rest-api-specs/issues/11129

**Comment by ozinus** [comment]
This is still an issue. Any update on this..?

**Comment by lordisp** [comment]
any updates on this one? I having the same issue

**Comment by TeplrGuy** [comment]
I think I may have found a solution @lordisp @mleziva 

set use_32_bit_worker_process to false.. https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/app_service#use_32_bit_worker_process

This will set it to 64 and default to php. I think 32 bit may not be available for php

**Comment by lordisp** [comment]
@TeplrGuy I'll skip this for couple of reasons

1.  PHP8 will no longer supported under windows app services: [PHP on App Service](https://github.com/Azure/app-service-linux-docs/blob/master/Runtime_Support/php_support.md#php-8)
2. virtual directory change is only supported on windows app service [configure common settings for an App Service](https://docs.microsoft.com/en-us/azure/app-service/configure-common#configure-path-mappings)
3. [Laravel application lifecycle](https://laravel.com/docs/8.x/lifecycle) begins in the public directory instead of the application's root directory. The default PHP Docker image for App Service uses Apache, and it doesn't let you customize the `DocumentRoot` for Laravel. Using `.htaccess` to rewrite all requests to point to /public instead of the root directory may works for developments, however, I don't like this approach for production.

Long story short: I move to AKS
